pyproject.toml

```diff
@@ -90,3 +90,99 @@ max-args=10
 
 [tool.pylint.'BASIC']
 good-names=["a", "b", "mu"]
+
+[tool.ruff]
+exclude = ["_version.py", "metasyn/distribution/legacy.*"]
+line-length = 100
+select = ["E", "F", "W", "I", "N"]
+
+[tool.ruff.pep8-naming]
+ignore-names = ["a", "b", "mu"]
+
+[tool.tox]
+legacy_tox_ini = """
+    [tox]
+    env_list =
+        py311
```

Comment:
I mean this env list. Doesn't it make sense to have also ruff/mypy/etc in there. I don't really know how it works, but I understood that then you can just run the tox command without "-e" to run all the tests.

Reply:
You're right! I've fixed this. You can now call them individually with -e, or run them all by simply calling tox.